client/src/app/health/route.ts

```diff
@@ -1,4 +1,25 @@
-// ? this endpoint is used as health check for the application
+import { env } from "../../env";
```

Comment:
I am not sure about this at all, but I believe FE magicians have another way to bring env values, that ensures validation and stuff. I might be wrong tho.

@atrincas what do you think?

Reply:
This is what they use.
The contents of the file:
`import { createEnv } from "@t3-oss/env-nextjs";
import { z } from "zod";

export const env = createEnv({
  /*
   * Serverside Environment variables, not available on the client.
   * Will throw if you access these variables on the client.
   */
  server: {
    NEXTAUTH_URL: z.preprocess(
      // This makes Vercel deployments not fail if you don't set NEXTAUTH_URL
      // Since NextAuth automatically uses the VERCEL_URL if present.
      (str) => process.env.VERCEL_URL ?? str,
      // VERCEL_URL doesnt include `https` so it cant be validated as a URL
      process.env.VERCEL ? z.string() : z.string().url().min(1),
    ),
    NEXTAUTH_SECRET: z.string().min(1),
    AUTH_CREDENTIALS: z.string().min(3).includes(":").optional(),
  },
  /*
   * Environment variables available on the client (and server).
   *
   * 💡 You'll get type errors if these are not prefixed with NEXT_PUBLIC_.
   */
  client: {
    NEXT_PUBLIC_API_URL: z.string().min(1).url(),
  },
  /*
   * Due to how Next.js bundles environment variables on Edge and Client,
   * we need to manually destructure them to make sure all are included in bundle.
   *
   * 💡 You'll get type errors if not all variables from `server` & `client` are included here.
   */
  runtimeEnv: {
    NEXTAUTH_URL: process.env.NEXTAUTH_URL,
    NEXTAUTH_SECRET: process.env.NEXTAUTH_SECRET,
    AUTH_CREDENTIALS: process.env.AUTH_CREDENTIALS,
    NEXT_PUBLIC_API_URL: process.env.NEXT_PUBLIC_API_URL,
  },
});
`

Reply:
Done